script/Ciel/Day 14/14_01B_01_CIEL14B_1B.txt

```diff
@@ -699,57 +792,65 @@ Suddenly, an enormous wall appears in front of us.
 [sha:0809cecca2f252e0902b2b0ef95228b4d41ca310]{
 -- Page 80, Offset 38578.
 -- それはどう見ても中世の城の壁面で、俺たちは空を跳躍している最中で、もう衝突するしか選択肢は存在しなかった。
-No matter how you look at it, the walls appears to belong to a medieval castle, and as we were in the middle of leaping through the air, there was no other option but to collide with that thing.
+Crazy at it sounds, there's no denying that it's the outer wall of some medieval castle,
+ and because we're mid-jump, there's nothing we can do to stop ourselves from careening straight into it.
 }
 [sha:bb7703aaeb1c508b7db656a428b79cf339aba6af]{
 -- Page 81, Offset 38579.
 -- 「っ……！　遠野くん、口を閉じて！」
-"...! Tohno-kun, hold your breath!"
+// I _assume_ this is the idea over 'shut up'?
+"%{i}Ah%{/i}! Tohno-kun, careful not to bite your tongue!"
 }
 [sha:5e8ea3684adaf855f5c808fbd2ec85b420d2ea21]{
 -- Page 82, Offset 38580.
 -- 先輩は俺の腕を掴んだまま、出現した壁を蹴りつけた。
-While grabbing onto my arm, Senpai steps onto the newly-formed wall.
+Senpai grabs me by the arm and slams heel-first into the newly-formed wall.
 }
 [sha:b57f862ae903a0fb8c99cb05c09feac5ffd449da]{
 -- Page 82, Offset 38581.
 -- タン、と硬い石を踏んでいく音がする。
-The sound echoes of footsteps on a hard stone surface.
+I hear her footsteps echoing off the hard stone surface.
 }
 [sha:e46a7a2d15c44f8eacbc1cb16d0c9590ea1ffa81]{
 -- Page 82, Offset 38582.
 -- 先輩は壁を足場にして、真横につっ走って回避した。
-Senpai uses the wall as a pavement and runs across it horizontally.
+Senpai is treating the wall like it's a road,
+ running sideways along it to avoid crashing into it.
```

Comment:
Not sure what the whole road thing is about.
>Using it as footing, Senpai runs sideways along the wall, avoiding the crash.

Reply:
Juxtaposition of the whole running along the wrong plane. I prefer it to the suggestion still.

Reply:
I sorta feel it, but it doesn't flow super smoothly.

Something like this would be fine

>Senpai uses the wall like a road, running sideways along it to avoid crashing.

also the "into it" just goes over the limit of one too many references to the same thing.